cmd/ak/cmd/projects/download.go

```diff
@@ -14,36 +15,34 @@ import (
 
 var outputDirectory string
 
-var downloadResourcesCmd = common.StandardCommand(&cobra.Command{
-	Use:   "download-resources <project name or ID> [--output-dir=...]",
-	Short: "Download the project's resources",
+var downloadCmd = common.StandardCommand(&cobra.Command{
```

Comment:
download a project is confusing, we might need it as download project definition or download project manifest or something